pyanaconda/ui/lib/payload.py

```diff
@@ -0,0 +1,58 @@
+#
```

Comment:
I think this code should be somewhere in the `/pyanaconda/payload` not UI code.

Reply:
`pyanaconda.payload` will be removed. We usually put these functions to `pyanaconda.core` when they are shared with modules or to `pyanaconda.ui.lib` when they are used only by the UI. Look at `pyanaconda.ui.lib.storage` and `pyanaconda.ui.lib.users`.

Reply:
It is true, that we might want to merge `pyanaconda.ui.lib` with `pyanaconda.core`.

Reply:
OK, thanks for explanation.